test/built-ins/BigInt/asIntN/bigint-tobigint-toprimitive.js

```diff
@@ -22,64 +22,64 @@ assert.sameValue(BigInt.asIntN(2, {
   },
   valueOf: err,
   toString: err
-}), 1n, "ToPrimitive: @@toPrimitive takes precedence");
+}), 1 n, "ToPrimitive: @@toPrimitive takes precedence");
```

Comment:
this should probably be reported as an issue of jsbeautify.

Reply:
see also https://github.com/tc39/test262/pull/1416#discussion_r167602054

Reply:
Sorry, I thought I found and fixed all of these!